src/modules/solution-explorer/solution-explorer-list/solution-explorer-list.ts

```diff
@@ -132,7 +118,7 @@ export class SolutionExplorerList {
       solutionExplorer = await this._solutionExplorerServiceFactory.newFileSystemSolutionExplorer();
     }
 
-    const identity: IIdentity = this._createIdentityForSolutionExplorer();
+    const identity: IIdentity = await this._createIdentityForSolutionExplorer();
```

Comment:
```suggestion
    const identity: IIdentity = this._createIdentityForSolutionExplorer();
```

Remove this await if the comment below on this called method is right